packages/botonic-cli/README.md

```diff
@@ -68,7 +64,7 @@ EXAMPLE
   🚀 Bot deployed!
 ```
 
-_See code: [src/commands/deploy.js](https://github.com/hubtype/botonic/blob/master/packages/botonic-cli/src/commands/deploy.ts)_
+_See code: [lib/commands/deploy.js](https://github.com/hubtype/botonic/blob/v0.13.0-rc.0/lib/commands/deploy.js)_
```

Comment:
I understand having v.13.0-rc instead of master is more accurate, but I don't think it's worth our effort updating it at each release (do other packages do it?)

Reply:
is it missing "packages/botonic-cli/" ?

Reply:
I cannot handle this directly, as this is done automatically by oclif